.github/CONTRIBUTING.md

```diff
@@ -64,6 +62,72 @@ To build containerized `moby/buildkit:local` and `moby/buildkit:local-rootless`
 make images
 ```
 
+### Run BuildKit
+
+You can launch the backend BuildKit daemon either directly or in a container:
+
+```bash
+# run the daemon directly
```

Comment:
Do we need to explicitly call out that this method won't work on macOS or Windows?

Reply:
Good point :tada: